server/src/test/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 quarkus.http.test-port=8083
-io.delta.sharing.api.server.defaultMaxResults=10
\ No newline at end of file
+io.delta.sharing.api.server.defaultMaxResults=10
+
+provider.aws.region=eu-west-1
+provider.aws.accessKey=${APP_AWS_ACCESS_KEY_ID}
+provider.aws.secretKey=${APP_AWS_SECRET_ACCESS_KEY}
```

Comment:
why are these needed?

Reply:
they are for tests only. I can highlight this